src/background.js

```diff
@@ -357,8 +357,27 @@ const rollout = {
   },
 };
 
+function studyCheck(study) {
```

Comment:
I think it would make sense to make this function more self contained and fetch the study from Normandy in this function instead of taking it as a parameter.